github/src/main/kotlin/gropius/sync/github/NodeSourcerer.kt

```diff
@@ -83,77 +83,124 @@ class NodeSourcerer(
     /**
      * Create an issuebody for the given issue is in the database
      * @param info The issue to created this body for
+     * @param imsProjectConfig Config of the active project
      * @return The finished body
      */
-    private suspend fun createIssueBody(info: IssueData): Body {
-        val user = ensureUser(info.author!!)
-        val issueBody = Body(
-            info.createdAt, OffsetDateTime.now(), (info as? IssueDataExtensive)?.body ?: "", info.createdAt
+    private suspend fun createIssueBody(imsProjectConfig: IMSProjectConfig, info: IssueData): Body {
+        val user = ensureUser(imsProjectConfig, info.author!!)
+        var issueBody = Body(
+            info.createdAt, info.createdAt, (info as? IssueDataExtensive)?.body ?: "", info.createdAt
         )
         issueBody.createdBy().value = user
         issueBody.lastModifiedBy().value = user
         issueBody.bodyLastEditedBy().value = user
+        issueBody = neoOperations.save(issueBody).awaitSingle()
         return issueBody
     }
 
+    /**
+     * Fills the body of the issue with the data from info and save the resulting issue
+     * @param imsProjectConfig the config to use while processing the issue
+     * @param info the data from the GitHub api
+     * @param issue the issue to start working with
+     * @return true if the content of the issue has been changed
+     */
+    private suspend fun fillIssueBodyAndSave(
+        imsProjectConfig: IMSProjectConfig, info: IssueData, issue: Issue
+    ): Boolean {
+        var needSave = false
```

Comment:
imho the name of this variable is a bit weird  
because without the outer context (which is now missing due to the refactor), it is unclear what needs to be saved
maybe change to something like `wasUpdated` or so

Reply:
a8aa7c67e5bdd83e0eb4e80675c219376f9691af